topics/icrc_25_signer_interaction_standard.md

```diff
@@ -94,20 +95,22 @@ While processing the request from the relying party, the signer can cancel it at
 
 #### Use-Case
 
-1. The relying party sends a `icrc25_request_permission` request to the signer.
+1. The relying party sends a `icrc25_request_permission` message to the signer.
 2. Upon receiving the message, the signer first checks if it can process the message.
     - If the request version is not supported by the signer, the signer sends a response with an error back to the relying party.
-3. Next, the signer presents the details of the to-be-established connection to the user and asks the user to select identities that will be paired in response. If the user has never interacted with this relying party before, the signer should display information explaining that the user is about to establish a connection with a new relying party.
-    > **Note:** The signer should maintain a list of relying parties that are trusted by the user. It is recommended that signers
-    assist users when deciding to grant permissions to new relying parties, e.g. by maintaining a list of well-known relying parties
-    and displaying additional information about the relying party, such as its name, logo, etc., or in the case of an unknown relying
-    party, by displaying a warning. 
-    - If the user approves the request, the signer saves information about the granted permission scopes and sends a successful response back to the relying party.
+3. The signer removes any unrecognized scopes from the list of requested scopes.
+4. Depending on the session state the signer either skips or displays the details of the to-be-established connection to the user:
+    - If there is no ongoing session with the relying party:
```

Comment:
nit: maybe instead having this line, and then "- If there already is an active session with the relying party.." below, have something like: 
- If there is an active session with the relying party, skip to the next step, otherwise:"